lms/services/lti_outcomes.py

```diff
@@ -176,16 +137,17 @@ def find_element(cls, xml_element, path):
         return xml_element.find(f".//{xpath}")
 
     @classmethod
-    def _send_request(cls, outcomes_request_params, pox_body):
+    def _send_request(cls, outcomes_request_params, request_body):
```

Comment:
`pox_body` is a bad name because there are two POX bodies. The one we send and the one we receive.